src/test/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/OffencesControllerIntTest.kt

```diff
@@ -177,16 +171,7 @@ class OffencesControllerIntTest : IntegrationTestBase() {
     assertThat(result.first { it.code == "XX99002" }!!.schedules)
       .usingRecursiveComparison()
       .ignoringFieldsMatchingRegexes(".*id")
-      .isEqualTo(
-        listOf(
-          ScheduleDetails(
-            act = "Criminal Justice Act 2003",
-            code = "15",
-            url = "https://www.legislation.gov.uk/ukpga/2003/44/schedule/15",
-            schedulePartNumbers = listOf(1)
-          )
-        )
-      )
+      .isEqualTo(emptyList<ScheduleDetails> ())
```

Comment:
Very minor. Can you not just use `.isEmpty`() here?

Reply:
yes youre right of course! good spot